src/Constraint/ArraySubset.php

```diff
@@ -10,6 +10,7 @@
 use PHPUnit\Framework\ExpectationFailedException;
 use PHPUnit\SebastianBergmann\Comparator\ComparisonFailure as Phar_ComparisonFailure;
 use SebastianBergmann\Comparator\ComparisonFailure;
+use SebastianBergmann\Exporter\Exporter;
```

Comment:
IIRC this will break when people use the PHPUnit PHAR file to run tests

Reply:
Might be interesting to find a way to run a test suite with phar in these cases right?

Reply:
I just added that (running the test against PHARs in CI) for the PHPUnit Polyfills too. You should be able to use that change as an example for how to set that up: https://github.com/Yoast/PHPUnit-Polyfills/pull/160

Also note that the latest PHPUnit releases contain a change in how the PHARs are build, so now you need to take three different potential class names into account for the `Exporter`.... See: https://github.com/Yoast/PHPUnit-Polyfills/pull/161 for how I solved this in the PHPUnit Polyfills.